src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.RegistrationServiceException;
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static final String INVALID_LOGIN = "LOGIN";
+    private static final String VALID_LOGIN = "LOGIN@GMAIL";
+    private static final String EXISTING_LOGIN = "LOGIN_OLEG";
+    private static final String INVALID_PASSWORD = "PASS";
+    private static final String VALID_PASSWORD = "PASSWORD";
+    private static final int INVALID_AGE = 15;
+    private static final int VALID_AGE = 20;
+    private static final User EXISTING = new User(EXISTING_LOGIN, VALID_PASSWORD, VALID_AGE);
+    private RegistrationService registrationService;
+    private StorageDao storageDao;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        storageDao.add(EXISTING);
+    }
+
+    @Test
+    void validRegistration() {
+        User user = new User(VALID_LOGIN, VALID_PASSWORD, VALID_AGE);
+        User registeredUser = registrationService.register(user);

```

Comment:
Your register method returns original user. You don't check if user was added to db